style.css

```diff
@@ -0,0 +1,453 @@
+  }
+
+  .up-section__description {
+    padding-left: 9px;
+  }
+
+  .up-section__btn {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    width: 490px;
+    padding-left: 5px;
+    margin-top: 23px;
+    text-decoration: none;
+    color: #000;
+  }
+
+  .up-section__btn .up-section__btn_offer {
+    position: relative;
+    width: 232px;
+    padding-top: 14px;
+  }
+
+  .up-section__btn_offer:hover {
+    transform: scale(1.1);
+  }
+
+  .up-section__btn .up-section__btn_offer::after {
+    position: absolute;
+    top: 17px;
+    right: -3px;
+    display: block;
+    content: '';
+    width: 44px;
+    height: 12px;
+    background-image: url(../img/vector-1.png);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .up-section__scrooll {
+    bottom: 39px;
+    width: 27px;
+    height: 46px;
+  }
+
+  .up-section__scrooll:hover {
+    transform: scale(1.05);
+  }
+
+  .up-section__contacts {
+    margin-top: 69px;
+    margin-bottom: 21px;
+    margin-left: 12px;
+  }
+
+  .social {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    width: 153px;
+    list-style: none;
+  }
+
+  .social__item:hover {
+    transform: scale(1.05);
+  }
+}
+
+@media (min-width: 375px) and (max-width: 992px) {
+  .header__head {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    max-width: 461px;
+    background: #fff;
+  }
+
+  .header__logo {
+    margin-top: 14px;
+    margin-left: 26px;
+    color: #000;
+  }
+
+  .up-section__title {
+    text-align: center;
+    margin-bottom: 46px;
+  }
+
+  .up-section__scrooll {
+    bottom: -60px;
+    width: 27px;
+    height: 46px;
+  }
+
+  .header__lang,
+  .up-section__btn, {
+    display: none;
+  }
+}
+
+@media (max-width: 375px) {
+  .header__head {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    max-width: 461px;
+    background: #fff;
+  }
+
+  .header__logo {
+    margin-top: 14px;
+    margin-left: 26px;
+    color: #000;
+  }
+
+  .header__lang,
+  .up-section__btn,
+  .up-section__scrooll {
+    display: none;
+  }
+
+  .up-section__title {
+    text-align: auto;
+    bottom: 66px;
+  }
+
+  .up-section__title::after {
+    position: absolute;
+    top: 120px;
+    left: 150px;
+    display: block;
+    content: '';
+    width: 14px;
+    height: 14px;
+    border: 3px solid #000;
+    border-right-color: transparent;
+    border-top-color: transparent;
+    -webkit-transform: rotate(-44deg);
+    -ms-transform: rotate(-44deg);
+    transform: rotate(-44deg);
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5676737)